README.md

```diff
@@ -54,17 +71,36 @@ Curation of papers is generally based on the recent publication in top AI confen
 - **Bayesian-optimized Bidirectional LSTM Regression Model for Non-intrusive Load Monitoring** [[link]](https://ieeexplore.ieee.org/document/8683110)
   - Maria Kaselimi, Nikolaos Doulamis, Anastasios Doulamis, Athanasios Voulodimos, Eftychios Protopapadakis. *ICASSP'19*
 
-### Others
+#### Others
 - **Gemello: Creating a Detailed Energy Breakdown from just the Monthly Electricity Bill** [[pdf]](https://www.kdd.org/kdd2016/papers/files/adp1036-batraA.pdf)
   - Nipun Batra, Amarjeet Singh, Kamin Whitehouse. *KDD'16*
 - **Disambiguating Energy Disaggregation: A Collective Probabilistic Approach** [[pdf]](https://www.ijcai.org/proceedings/2017/0398.pdf)
   - Sabina Tomkins, Jay Pujara, Lise Getoor. *IJCAI'17*
+- **On the Applicability of Correlation Filters for Appliance Detection in Smart Meter Readings** [[pdf]](http://mobile.nes.aau.at/publications/klemenjak-2017-Correlation_Filters.pdf)
+  - C Klemenjak, W Elmenreich - 2017 IEEE International Conference on Smart Grid …, 2017
 - **Matrix Factorisation for Scalable Energy Breakdown** [[pdf]](http://www.aaai.org/ocs/index.php/AAAI/AAAI17/paper/download/14885/14054)
   - Nipun Batra, Hongning Wang, Amarjeet Singh, Kamin Whitehouse. *AAAI'17*
 - **Transferring Decomposed Tensors for Scalable Energy Breakdown across Regions** [[pdf]](https://aaai.org/ocs/index.php/AAAI/AAAI18/paper/view/16858/15754)
   - Nipun Batra, Yiling Jia, Honging Wang, Kamin Whitehouse. *AAAI'18*
 
+### Papers on Performance Evaluation
+
+
+- **Nonintrusive load monitoring (NILM) performance evaluation** [[pdf]](https://link.springer.com/article/10.1007/s12053-014-9306-2)
+  - S Makonin, F Popowich. Energy Efficiency, 2015
+- **Performance evaluation in non-intrusive load monitoring: Datasets, metrics, and tools-A review** [[pdf]](https://www.researchgate.net/publication/325294847_Performance_evaluation_in_non-intrusive_load_monitoring_Datasets_metrics_and_tools-A_review)
+  - L Pereira, N Nunes. Wiley Interdisciplinary Reviews: Data Mining and …, 2018
+
 ## Public Datasets
+
+### Papers on Datasets
+
+- **Position Paper: Electricity Consumption Data Sets: Pitfalls and Opportunities** [[pdf]](http://makonin.com/doc/BuildSys_2019.pdf)
```

Comment:
I think there are a lot of discussions about NILM datasets. Are there any other papers?

Reply:
Indeed, there are many papers on energy datasets for NILM. However, very few of them actually talk about data issues and requirements for NILM or the big discrepancy between NILM datasets. I think this issue has big implications on NILM scholarship and should be explored in future.

I found this paper on NILM data requirements recently, that could fit to this topic:

https://www.mdpi.com/1996-1073/12/9/1696/pdf

also, Nalmpantis talks about such issues in his paper:

https://intelligence.csd.auth.gr/wp-content/uploads/2019/03/Nalmpantis-AIR-2018.pdf